scripts/run_app.sh

```diff
@@ -50,6 +49,8 @@ if [ ! -s "static" ]; then
   exit 1
 fi
 
+$DIR/run_translate_schemas.sh
```

Comment:
The command will need to be added to the Jenkins build, similar to the yarn compile we do. It might be worth having a `run_build.sh` script that does both (yarn compile and translate schemas) though?